templates/schedule.html

```diff
@@ -0,0 +1,137 @@
+<h2>Schedule</h2>
+
+<p>Please note that all times are CST.</p>
+
+<table style="border-spacing: 2px; border-collapse: separate">
```

Comment:
Add fixed column widths and column borders to make table more readable

Reply:
I tried column bordes in a first attempt but found that they made the table less readable. About the column widths: Do you have something specific in mind or do you want towards having more space between adjacent cells?

Reply:
My main intention is to make the difference between all-track timeslots (like breakfast) and track-specific ones (like »hack on«/»workshop«) more obvious.

Reply:
As discussed via mail.